### PR TITLE
Product Specifc View

### PR DESCRIPTION
**Pull request by bob-zs**
Dev link: http://localhost:3000/#/products/3

Screen Shot 2022-08-21 at 6 41 11 PM

### CODE REVIEW

**Comment by bob-zs** [comment]
The description comes after the "Add to Cart". Should it come before?

File: src/Products/ProductSpecificView.js [hunk 1]
```diff
@@ -1,75 +1,34 @@
...
+const ProductSpecificView = ({ product , addToCart, lineItems }) => {
+    return (
+        <div className="product-specific-view">
+            <div className="product-specific-view-img">
+                <img src={ product.imgUrl } width='240' height='160'/>
+            </div>
+            <div>
+                <AddCartHelper addToCart={ addToCart } product={ product } lineItems={ lineItems } />
+                { product.description }<br />
+            </div>
```
Comment (bob-zs) [inline src/Products/ProductSpecificView.js:15]: `<AddCartHelper>` component before description ok?

**Comment by bob-zs** [comment]
Also, should there be more to this? This was a lot simpler when I moved it to a functional component?

**Review by alejandro-loja — APPROVED** [review]
Thanks

**Pull request merged**